notebooks/117-optimize-preprocessing/117-optimize-preprocessing.ipynb

```diff
@@ -0,0 +1,498 @@
+{
```

Comment:
<p>I think, it is better use python style of class naming for avoid misleading.</p><p> </p><p>Also I think read from disk is not suitable clarification here, can we remove it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/811/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
done